### ISSUE TITLE
Stop iOS accessibility Voiceover to access items that are not rendered

### ISSUE THREAD

**Issue by shadowcatx** [aksonov/react-native-router-flux#1506]
Hi, we have been using RNRF for navigation and it works perfectly until we have to cater for accessibility. We are on iOS, and the issue is that VoiceOver can detect items from previous scene that is no longer visible. 

I have spent some time digging into how this can be resolved, but failed to find a method. Please let me know if there is a way to do this. If this is by design not achievable please let me know too, thanks.

### Version
- react-native-router-flux v3.35.0
- react-native v0.39.2

### Expected behaviour
I expect only the clickable area on the current Scene would be accessible. 

### Actual behaviour
Items from previous scenes can be accessed. While actions will not be triggered, the items can be selected and the label would be read.

### Steps to reproduce

1. Create two scenes, A and B, with RNRF. Place a clickable button on scene A.
2. Turn on accessibility VoiceOver
3. Navigating from scene A to scene B. All items on Scene A should be gone. Now click on the empty place in scene B where the button on scene A used to be, the button would be selected and read out by VoiceOver.

**Comment by saikrishna321** [comment]
@aksonov can you pls help on this, even when we run the automated tests we get the previous screen elements on the DOM.

**Comment by jonolock91** [comment]
Was there any resolution to this issue?

**Comment by aksonov** [comment]
Sorry, but RNRF doesn't work with VoiceOver - it is responsibility of underlying implementation. Check v4, if it is still issue, report to react-navigation package.

**Comment by jonolock91** [comment]
Is this issue to do with React Native itself then?
